torch/csrc/cuda/shared/cudart.cpp

```diff
@@ -92,7 +92,7 @@ void initCudartBindings(PyObject* module) {
       [](uintptr_t ptr) -> cudaError_t {
         return C10_CUDA_ERROR_HANDLED(cudaStreamDestroy((cudaStream_t)ptr));
       });
-#if !defined(USE_ROCM)
+#if !defined(USE_ROCM) && CUDA_VERSION < 12000
```

Comment:
does `cudaProfilerInitialize` not work anymore?

Reply:
Thanks! Just add some fix.